images/README.md

```diff
@@ -1,21 +1,35 @@
 # images
 
-To help us build an image reliably, we are relying on `kiwi` and `microOS`.
-
-In `microOS/` one will be able to find `config.xml`, describing the image to be
-created, with several different possible profiles. This file has been obtained
-from [openSUSE's MicroOS Tumbleweed OBS repository](https://build.opensuse.org/package/show/openSUSE:Factory/openSUSE-MicroOS),
-and slightly adapted to our needs.
-
-One will also be able to find, under the same directory, a `config.sh` script.
-This is automatically run by `kiwi` during the build process, at some point
-after the packages have been installed, and before the image is generated. This
-is generally where services will be enabled, for example. We are, at the moment
-of writing, simply installing additional packages that are not available through
-the repositories, relying on `pip` to do so. In the future, we expect to use
-this file to enable the backend's service.
-
-One can also find a `build.sh` script, which is merely a helper to build a
-microOS image from `config.xml`. At the moment, a Vagrant image will be
-generated. Feel free to run `build.sh --help` for slightly more info, but the
-script itself should be fairly straightforward.
+To help us build an image reliably, we are relying on `kiwi`. We do so by
+offering an image to natively build using `kiwi`, and containerized
+environments to build this image on.
+
+## Aquarium's image
+
+In `aquarium/` one will be able to find `config.xml`, describing the image to
+be created, with several different possible profiles. This file was initially
+based on [openSUSE's MicroOS Tumbleweed image](https://build.opensuse.org/package/show/openSUSE:Factory/openSUSE-MicroOS),
+but evolved to become its own thing, focused solely on our use cases and needs.
+
+One will also be able to find, under the same directory, a couple of scripts:
+`config.sh` and `disk.sh`. Both will be executed by `kiwi` at different steps of
+the [build process](https://osinside.github.io/kiwi/concept_and_workflow/shell_scripts.html).
+
+The former, `config.sh`, will be run at some point after the packages have
+been installed, and before the image is generated. During this step we will be
+installing additional packages not available through our repositories, namely
+python packages through `pip`, and will enable the `aquarium` service, which
+will be then be started on boot.
+
+The latter, `disk.sh`, is run within a chrooted image mount, but before the
+image is finalized. During this step we will be obtaining container images
+needed for Aquarium's execution, so we have them available upon first run,
+including an image for `Ceph` and an image for `etcd`.
+
+
+## Containerized build environment
+
+Under `builder/` we provide a `Dockerfile` that will be able to automatically
+build an image from a checked out branch. This might be useful for those on
+distributions that do not support `kiwi`, or under which `kiwi` does not work
+reliably.
```

Comment:
My opinion is to make this the only way we build images so that we aren't maintaining multiple environments. (Can be a separate change though).

Reply:
Agreed on both counts